lib/rbs/prototype/runtime.rb

```diff
@@ -399,6 +404,14 @@ def generate_module(mod)
 
         generate_constants mod
       end
+
+      def const_name(const)
+        method = const.method(:name)
+        until method.owner == Module
+          method = method.super_method
+        end
+        method.call
+      end
```

Comment:
There may be a more smart way... 🤔

Reply:
What `setup.rb` is doing would help.

* https://github.com/ruby/rbs/blob/master/lib/rbs/test/setup.rb#L58-L61

Reply:
@soutaro Thank you for the help! I've fixed it via a3509ad.

Reply:
I've added caching a return value of `Module.instance_method(:name)` in order to avoid extra object creation.
But this may be overthinking. If you consider it needless, I will remove the caching.